packages/ag-charts-enterprise/src/series/treemap/treemapSeries.test.ts

```diff
@@ -51,15 +51,7 @@ describe('HierarchyChart', () => {
             expect(console.warn).not.toBeCalled();
         });
 
-        const SIMPLIFIED_EXAMPLE = {
-            ...GALLERY_EXAMPLES.MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE.options,
-            data: [
-                {
-                    ...GALLERY_EXAMPLES.MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE.options.data[0],
-                    children: GALLERY_EXAMPLES.MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE.options.data[0].children.slice(0, 1),
-                },
-            ],
-        };
+        const SIMPLIFIED_EXAMPLE = GALLERY_EXAMPLES.MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE.options;
```

Comment:
Nit: I think it would be good to keep the data-set the same as before, so we can see the visual differences between the previous and new implementations?

(Happy to change this as a follow-up though)